exonum/src/api/node/public/explorer.rs

```diff
@@ -171,6 +172,80 @@ impl AsRef<[u8]> for TransactionHex {
     }
 }
 
+/// Status of a call.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CallStatusResponse {
```

Comment:
Couple of notes here:

- `status` is redundant because it is contained in the proof anyway
- It could make sense to return the `BlockProof` (block header with precommits), at least if the transaction is queried by hash (in this case, the caller may reasonably not have `BlockProof` locally)